src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Controllers/ManageApprenticesController.cs

```diff
@@ -1,21 +1,34 @@
-﻿using System.Threading.Tasks;
+﻿using System.Linq;
+using System.Threading.Tasks;
 using Microsoft.AspNetCore.Mvc;
+using SFA.DAS.Commitments.Shared.Interfaces;
 using SFA.DAS.ProviderCommitments.Web.Models;
 
 namespace SFA.DAS.ProviderCommitments.Web.Controllers
 {
-    [Route("v2/{providerId}/apprentices/manage")]
+    [Route("v2/{providerId}/apprentices")]
     public class ManageApprenticesController : Controller
     {
+        private readonly ICommitmentsService _commitmentsService;
+
+        public ManageApprenticesController(ICommitmentsService commitmentsService)
+        {
+            _commitmentsService = commitmentsService;
+        }
+
         [Route("all")]
-        public IActionResult Index(long providerId)
+        public async Task<IActionResult> Index(long providerId)
```

Comment:
unless providerId needs to be long elsewhere i've change this to a uint as the binder can then enforce incorrect values even before they get to your code